compose/.apps/filebrowser/filebrowser.labels.yml

```
@@ -0,0 +1,13 @@
+services:
+  filebrowser:
+    labels:
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.description: "Filebrowser provides a file managing interface within a specified directory"
+      com.dockstarter.appinfo.nicename: "Filebrowser"
+      com.dockstarter.appvars.filebrowser_base_url: ""
+      com.dockstarter.appvars.filebrowser_database: "/config/database.db"
+      com.dockstarter.appvars.filebrowser_enabled: "false"
+      com.dockstarter.appvars.filebrowser_log: "/config/filebrowser.log"
+      com.dockstarter.appvars.filebrowser_network_mode: ""
+      com.dockstarter.appvars.filebrowser_port_80: "80"
+      com.dockstarter.appvars.filebrowser_share_dir: "~/.config/appdata/filebrowser"
```

```suggestion
      com.dockstarter.appvars.filebrowser_share_dir: "~/.config/appdata/filebrowser/share"
```
I think this should be a subdirectory to keep it separated from the actual config and not imply that it has to be the same folder as the config.

---

Fair enough, I just filled that in so it wouldn't fail, but the end user needs to change it to the folder he wants to actually share anyways since that's the path that gets mounted to `/srv`